client/checkout/blocks/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -174,14 +169,23 @@ const PaymentProcessor = ( {
 					) {
 						return {
 							type: 'error',
-							message:
+							message: __(
 								'This payment method cannot be saved for future use.',
+								'woocommerce-payments'
+							),
 						};
 					}
 
-					await validateElements( elements );
```

Comment:
Added a `try`/`catch` block around `validateElements`. This function throws an exception, which was not captured before.